iterativeTrim.py

```diff
@@ -195,6 +197,26 @@ def writeChConfig(chConfig,vfat,vfatID,trimDAC,trimPol,mask,maskReason):
     vfatBoard = HwVFAT(cardName, args.link, args.debug)
     print("Opened connection")
 
+    if args.vfatConfig is not None:
+        try:
+            print 'Configuring VFAT Registers based on %s'%args.vfatConfig
+            vfatTree = r.TTree('vfatTree','Tree holding VFAT Configuration Parameters')
+            vfatTree.ReadFile(args.vfatConfig)
+            
+            for event in vfatTree :
+                # Skip masked vfats
+                if (args.vfatmask >> int(event.vfatN)) & 0x1:
+                    continue
+                    
+                # Write CFG_THR_ARM_DAC
+                print('Set link %d VFAT%d CFG_THR_ARM_DAC to %i'%(args.link,event.vfatN,event.vt1))
+                vfatBoard.setVFATThreshold(chip=int(event.vfatN), vt1=int(event.vt1))
+        except IOError as e:
+            print '%s does not seem to exist or is not readable'%options.filename
```

Comment:
Make sure all added `print` statements will be `python3` compatible, e.g., `print()` vs `print`
If the rest of the file being modified uses the `"{}".format()` style formatting, rather than `"%x"%val` style, also conform to that

Reply:
done